kiosk/src/Kiosk.css

```diff
@@ -1,98 +1,235 @@
+/* -------- GLOBAL-ISH SYTLING -------- */
+/* ----------------------------------- */
 html, body {
     height: 100%;
     width: 100%;
     padding: 0;
     margin: 0;
     overflow: hidden;
-    background-color: black;
+    background: linear-gradient(rgba(19, 69, 90, 0.81), rgb(5, 5, 5));
     font-family: 'Press Start 2P';
     color: white;
-    align-items: center;
-}
-
-.bg1 {
-    background: radial-gradient(62.83% 55.51% at 50% 50%, rgba(19, 69, 90, 0.81) 0%, #13455A 100%);
-    height: 100%;
-    width: 100%;
-}
-
-.background {
-    box-sizing: border-box;
-    height: 100%;
-    width: 100%;
-    opacity: 0.2;
-    border: 1px solid #141414;
-    filter: blur(137px);
 }
 
 .content {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
+    width: 100vw;
+    height: 100vh;
 }
 
 h1 {
     font-weight: 400;
-    font-size: 54px;
-    line-height: 54px;
+    font-size: 3.375em;
+    line-height: 3.375rem;
     text-align: center;
 }
 
 h2 {
     font-weight: 400;
-    font-size: 44px;
-    line-height: 44px;
+    font-size: 2.75em;
+    line-height: 2.75rem;
     text-align: center;
-    color: white;
 }
 
-.mainMenuHighScoreHeader {
+.sim-embed {
+    width: 100%;
+    height: 100%;
+    top:0;
+    left: -2vw;
+    position: absolute;
+}
+
+.footer {
+    position: fixed;
+    /* These numbers are relative to the size of the footer.png logo */
+    height: 5vh;
```

Comment:
avoid using the vh and vw units. use percentage instead

Reply:
Is it because vh and vw are based off of percentages? In projects in the past, I've had problems when all the units for containers were percents.

Reply:
I see that changing vw/vh to percentages doesn't change anything visually, but I'm curious what the benefit is underneath. I know percent is based off of the parent element while vw/vh is based off of the viewport (unless that is where my misunderstanding lies). If that's the case, wouldn't it be valuable to have larger divs/containers have vw/vh as the units so they do change according to the size of the screen and then have child elements use percentages to scale with its container? 

I know percentages allow for the same thing to happen, so I guess I'm just questioning the level of verbosity/readability for styling to solely use percentages. If you use percentages for all sizing, is it a bit harder to debug because you have to keep looking for what percent that element's size is based off of?

Reply:
it's mostly just a stylistic choice, most of our code uses percentages. I also find vw/vh to make css classes a little more difficult to reuse in responsive designs